weblab/entities/tests/test_views.py

```diff
@@ -43,6 +49,15 @@ def fake_upload_path(settings, tmpdir):
     return settings.MEDIA_ROOT
 
 
+def add_version(entity):
+    """Add a single commit/version to an entity"""
+    entity.init_repo()
+    in_repo_path = str(entity.repo_abs_path / 'entity.txt')
```

Comment:
What's happening here? Has the `/` operator overloaded?

Reply:
This is [pathlib](https://docs.python.org/3/library/pathlib.html), new in Python 3.4.

Reply:
Thanks! That looks pretty cool. Wonder if anyone will backport it...